iOS/Views/MovieTableViewCell.swift

```diff
@@ -0,0 +1,42 @@
+import UIKit
+import Shared
+
+class MovieTableViewCell: UITableViewCell {
+
+    @IBOutlet weak var movieBackdrop: UIImageView!
+    @IBOutlet weak var movieGenresImages: UIStackView!
+    @IBOutlet weak var movieReleaseDate: UILabel!
```

Comment:
Don't you think that the variable movieReleaseDate seems to be a Date instead of UILabel ?